UnityProject/Assets/Scripts/Items/Implants/Organs/Brain.cs

```diff
@@ -164,5 +176,33 @@ public void SetCannotSpeak(bool inValue)
 				}
 			}
 		}
+
+		#region Mind_stuff
+
+		public GameObject GameObject => gameObject;
+
+		public IPlayerPossessable Possessing { get; set; }
+
+		public GameObject PossessingObject { get; set; }
+
+		public Mind PossessingMind { get; set; }
+
+		public IPlayerPossessable PossessedBy { get; set; }
+
+		public MindNIPossessingEvent OnPossessedBy  { get; set; }
+
+		public Action OnActionEnterPlayerControl { get; set; }
+
+		public RegisterPlayer CurrentlyOn { get; set; }
+		bool IItemInOutMovedPlayer.PreviousSetValid { get; set; }
```

Comment:
did you intend to type ``public`` instead of ``bool``here?

Reply:
nah, that's correct

Reply:
why did you need to explicitly name the interface?

Reply:
also use explicit access modifier, ``private`` in this case.

Reply:
because it is only accessed by the interface

Reply:
![image](https://user-images.githubusercontent.com/27101670/213764123-1e825b3b-2b05-4294-b58b-c7e6d520ad37.png)

Reply:
aah, yeah you are typing in the interface. Everything should be public there.

Reply:
you can make it private, by not adding anything